src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -50,6 +62,9 @@ public class TenantServiceImpl implements TenantService {
   private final PermissionUserService permissionUserService;
   private final UserService userService;
   private final FolioModuleMetadata folioModuleMetadata;
+  private final UsersClient usersClient;
+  private final UserTenantService userTenantService;
+  private final KafkaService kafkaService;
```

Comment:
I think better to introduce new service for creating primary affiliation, it will simplify TenantService code, because we also add some new functionality here in future.
But I a ok also to leave as it

Reply:
done